cmd/openshift-install/create.go

```diff
@@ -217,3 +221,14 @@ func destroyBootstrap(ctx context.Context, directory string) (err error) {
 	logrus.Info("Destroying the bootstrap resources...")
 	return destroybootstrap.Destroy(rootOpts.dir)
 }
+
+// logComplete prints info upon completion
+func logComplete(directory string) error {
+	absDir, err := filepath.Abs(directory)
+	if err != nil {
+		return err
+	}
+	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s/auth/kubeconfig' to manage your cluster.", absDir)
```

Comment:
Did you not want [`filepath.Join`][1] to assemble the path with the local slash direction?

[1]: https://github.com/openshift/installer/commit/ac3c3aadb31b82515d24dbc9f6e6666f16509cf6#diff-266f03122200ace2d21f2e449b6a5503R231

Reply:
I just saw that from your branch, got it, added it lol

Reply:
:+1:  thanks :)